src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -863,6 +863,25 @@ public int ConnectionLifetime
         }
         int _connectionLifetime;
 
+        /// <summary>
+        /// Whether string value caching should be used.
+        /// </summary>
+        [Category("Pooling")]
+        [Description("Whether string value caching should be used.")]
+        [DisplayName("String caching")]
+        [NpgsqlConnectionStringProperty]
+        [DefaultValue(true)]
+        public bool StringCaching
```

Comment:
That's what I would like to avoid. If string caching gives a valuable benefit then I see no reason for that option at all.

Reply:
Removed